sphinx_gallery/tests/test_gen_gallery.py

```diff
@@ -22,6 +22,9 @@ def test_bad_config():
     sphinx_gallery_conf = dict(example_dir='')
     with pytest.raises(ConfigError, match="example_dir.*did you mean 'examples_dirs'?.*"):  # noqa: E501
         _complete_gallery_conf(sphinx_gallery_conf, '', True, False)
+    sphinx_gallery_conf = dict(builder='dirhtml')
+    with pytest.raises(ConfigError, match=".*dirhtml.*sphinx_gallery does not work.*"):  # noqa: E501
+        _complete_gallery_conf(sphinx_gallery_conf, '', True, False)
```

Comment:
This looks like it's not the right place to add a test actually. It's not a SG config thing, it's a sphinx builder thing. So I think this needs to be in a different test... let me look and push a quick commit to fix.